src/publisher/views/publish/dimension-chooser.jsx

```diff
@@ -62,10 +62,14 @@ export default function DimensionChooser(props) {
                       value: 'Date',
                       label: props.t('publish.dimension_type_chooser.chooser.date')
                     },
-                    {
-                      value: 'Geog',
-                      label: props.t('publish.dimension_type_chooser.chooser.geography')
-                    },
+                    ...(false
+                      ? [
+                          {
+                            value: 'Geog',
+                            label: props.t('publish.dimension_type_chooser.chooser.geography')
+                          }
+                        ]
+                      : []),
```

Comment:
I would just delete this tbh cause it's easy enough to copy from the lines above... who knows if/when this ever comes back